indicators.py

```diff
@@ -0,0 +1,41 @@
+import urllib.request
+import json
+
+apikey = "R1OZ339ORWVQ05E9"
+
+class Indicators: 
+
+    def __init__(self,ticker,date):
+        self.ticker = ticker
+        self.date = date
+        self.fiftydayaverage = self.get_moving_average_50()
+        self.twodayaverage = self.get_moving_average_200()
+        self.stock_prediction = self.stock_prediction()
+
+
+    def get_moving_average_50(self): 
+        url = "https://www.alphavantage.co/query?function=SMA&symbol="+self.ticker+"&interval=daily&time_period=50&series_type=low&apikey="+apikey
+        data = json.load(urllib.request.urlopen(url))
+        return data["Technical Analysis: SMA"][self.date]["SMA"]
+
+    def get_moving_average_200(self): 
```

Comment:
Could modify current `rolling_avg` method in `Stock.py` to fit both this and the `get_moving_average_50` into one function